### DISCUSSION TITLE
[Mobile] Reviewing PRs and 'File-Viewed'-Toast

### DISCUSSION THREAD

**Discussion by MauriceArikoglu** [community/community#40997] [Mobile]
Hi there,

The 'viewed file'-confirmation toast is blocking the review button. 
I suggest either
- moving it to the left 
- moving it up a bit
- moving it to the top

Screenshot for reference, toast can be seen blocking the review button on the bottom:
![image](https://user-images.githubusercontent.com/5528623/206145294-3a2744c1-c4d8-460e-a55f-a3fef796133e.png)

**Comment by candyho** [comment]
Hey! Thanks for the feedback. I will defer this feedback to the team and hopefully we can work on a fix soon!
